_component_design/tables-basic.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Tables (old)
+title: Tables - Basic
```

Comment:
file name can stay the same, but the title should be "Basic Tables"

Reply:
updated.